packages/boilerplate/api-client/tsconfig.json

```diff
@@ -1,5 +1,20 @@
 {
   "extends": "../tsconfig.base.json",
   "include": ["src"],
-  "exclude": ["node_modules", "lib"]
+  "exclude": ["node_modules", "lib"],
+  "compilerOptions": {
+    "outDir": "./lib",
+    "esModuleInterop": true,
+    "target": "ES5",
+    "module": "ES2015",
```

Comment:
What is the reason for using `target: 'ES5'` and `module: 'ES2015'`? We don't support legacy browsers (ES5) and at this stage I think it's safe to use ES2017

Reply:
Updated :)